client/my-sites/site-settings/stats.jsx

```diff
@@ -0,0 +1,186 @@
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import SectionHeader from 'components/section-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import QuerySiteRoles from 'components/data/query-site-roles';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteRoles } from 'state/site-roles/selectors';
+import {
+	isJetpackModuleActive,
+	isJetpackModuleUnavailableInDevelopmentMode,
+	isJetpackSiteInDevelopmentMode
+} from 'state/selectors';
+
+class SiteStats extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+		fields: {}
+	};
+
+	static propTypes = {
+		handleAutosavingToggle: PropTypes.func.isRequired,
+		setFieldValue: PropTypes.func.isRequired,
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+		fields: PropTypes.object,
+	};
+
+	onChangeToggleGroup = ( groupName, fieldName ) => {
+		return () => {
+			const { setFieldValue } = this.props;
+			let groupFields = this.getCurrentGroupFields( groupName );
+
+			if ( groupFields.includes( fieldName ) ) {
+				groupFields = groupFields.filter( field => field !== fieldName );
+			} else {
+				groupFields.push( fieldName );
+			}
+
+			setFieldValue( groupName, groupFields, true );
+		};
+	};
+
+	getCurrentGroupFields( groupName ) {
+		const { fields } = this.props;
+
+		if ( ! fields[ groupName ] ) {
+			return [];
+		}
+		return fields[ groupName ];
+	}
+
+	renderToggle( name, label, checked = null, onChange = null ) {
+		const {
+			fields,
+			handleAutosavingToggle,
+			isRequestingSettings,
+			isSavingSettings,
+			moduleUnavailable,
+			statsModuleActive
+		} = this.props;
+
+		if ( checked === null ) {
+			checked = !! fields[ name ];
+		}
+
+		if ( onChange === null ) {
+			onChange = handleAutosavingToggle( name );
+		}
+
+		return (
+			<CompactFormToggle
+				checked={ checked }
+				disabled={ isRequestingSettings || isSavingSettings || moduleUnavailable || ! statsModuleActive }
+				onChange={ onChange }
+				key={ name }
+			>
+				{ label }
+			</CompactFormToggle>
+		);
+	}
+
+	render() {
+		const {
+			siteId,
+			siteRoles,
+			translate
+		} = this.props;
+		const header = translate( 'Collecting valuable traffic stats and insights' );
+
+		return (
+			<div className="site-settings__traffic-settings">
+				<QueryJetpackConnection siteId={ siteId } />
+				<QuerySiteRoles siteId={ siteId } />
+
+				<SectionHeader label={ translate( 'Site stats' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'Site Stats' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-site-stats-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).